projects/angular-cld/src/lib/cloudinary-configuration.class.ts

```diff
@@ -23,4 +23,5 @@ export default interface CloudinaryConfiguration {
     readonly use_root_path?: boolean;
     readonly version?: string;
     readonly client_hints?: boolean;
+    readonly urlAnalytics?: boolean;
```

Comment:
We decided on urlAnalytics since this is an indicator in core in the cloudinary_url method if to append the param or not. Its the same option in core as well.

Reply:
No problem, just align the other places that says `SDKAnalytics`.